Polyfills/WebSocket/Source/WebSocket.cpp

```diff
@@ -211,14 +225,17 @@ namespace Babylon::Polyfills::Internal
 
     void WebSocket::MessageCallback(const std::string& message)
     {
-        m_runtimeScheduler([this, message]() {
+        m_runtimeScheduler([this, message, cancellationSource{m_cancellationSource}]() {
             try
             {
                 if (!m_onmessage.IsEmpty())
                 {
                     Napi::Object messageEvent = Napi::Object::New(Env());
                     messageEvent.Set("data", message);
-                    m_onmessage.Call({messageEvent});
+                    if (!cancellationSource->cancelled())
```

Comment:
Do the same thing as earlier and check at the beginning of the lambda.

Reply:
Fixed